frontend/app/models/attendance.js

```diff
@@ -4,11 +4,11 @@ import moment from 'moment'
 import Model  from './model'
 
 const { computed } = Ember
-const { attr, belongsTo, hasMany } = DS
+const { attr, belongsTo } = DS
 
 export default Model.extend({
   user:       belongsTo('user', { async: true })
-, activities: hasMany('activity')
+//, activities: hasMany('activity')
```

Comment:
Not that much. It's still working, but not usable. But I didn't care much because we were going to completely change the component anyway, I think.